onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -165,10 +168,11 @@ private Long checkArrivals(ObaArrivalInfoResponse response, Cursor c) {
                     depart = info.getScheduledArrivalTime();
                 }
 
-                return depart;
+                ArrivalInfo arrivalInfo = new ArrivalInfo(mContext, info, 0l, false);
```

Comment:
I think `ArrivalInfo arrivalInfo = new ArrivalInfo(mContext, info, 0l, false);` should be `ArrivalInfo arrivalInfo = new ArrivalInfo(mContext, info, response.getCurrentTime(), false);`?  Any reason not to pass in the current time here?  If now, I'd prefer to construct this with actual data, in case someone comes along later and tries to use one of the methods that depends on the current time.